pep/app/styles/_variables-theme.scss

```diff
@@ -30,3 +24,13 @@ $headings-font-family: Muli, -apple-system, BlinkMacSystemFont, 'Segoe UI', Robo
 $hr-border-color: rgba(#000, 0.06);
 $dropdown-border-color: #dbdbdb;
 $dropdown-border-radius: 0.21rem;
+// ---------------------------
+// Addon overrides
+// ---------------------------
+$button-spinner-loading-color1: rgba(#fff, 0.75);
```

Comment:
These seem like the same thing that was put into `tomato`. So Im assuming you can either delete one of these, or you would need to add these to all the other themes